tests/benchmark/compute/instruction/test_storage.py

```diff
@@ -33,36 +34,25 @@ def test_tload(
     val_mut: bool,
 ) -> None:
     """Benchmark TLOAD instruction."""
-    start_key = 41
-    code_key_mut = Bytecode()
-    code_val_mut = Bytecode()
     setup = Bytecode()
     if key_mut and val_mut:
-        setup = Op.PUSH1(start_key)
-        attack_block = Op.POP(Op.TLOAD(Op.DUP1))
-        code_key_mut = Op.POP + Op.GAS
-        code_val_mut = Op.TSTORE(Op.DUP2, Op.GAS)
+        setup = Op.GAS + Op.TSTORE(Op.DUP2, Op.GAS)
```

Comment:
While trying to read this code, I realize that `key_mut` -> `fixed_key`, `val_mut` -> `fixed_value`, and flipping the logic, would make it easier to read IMO.